### PR TITLE
fixed #180

### LINKED ISSUE #180: Email validation is not Done properly and it must be Done on backend also because  HTML,js can be edited too
image

### PR DESCRIPTION
**Pull request by engineerscodes**
Fixed Email validation filed

TEST CASE-
image
RES-
image
2)
image
RES-
image
3)
image
RES-
image
4)Valid one
image
RES-
image

### CODE REVIEW

**Review by nlok5923 — CHANGES_REQUESTED** [review]
Thanks for the pr i had left some comments PTAL

Also please fill pr description properly and always remember to link corresponding issue which this pr aims to solve.

**Comment by engineerscodes** [comment]
@nlok5923  Fixed, IT was due to code semantics i fallow ......vscode removes unwanted space and tab.

File: sign-up.html [hunk 1]
```diff
@@ -171,13 +171,7 @@
 
 
     </div>
-    <script>
-    const chk = document.getElementById('chk');
 
-chk.addEventListener('change', () => {
```
Comment (nlok5923) [inline sign-up.html:177]: @engineerscodes this changes aren't mentioned in the issue corresponding to this pr so please revert it.

File: login.html [hunk 2]
```diff
@@ -129,13 +129,7 @@
 
 
     </div>
-<script>
-    const chk = document.getElementById('chk');
 
-chk.addEventListener('change', () => {
```
Comment (nlok5923) [inline login.html:135]: same here

**Review by nlok5923 — CHANGES_REQUESTED** [review]
Thanks PTAL.

File: sign-up.html [hunk 3]
```diff
@@ -200,8 +200,9 @@
             var mob_num = document.getElementById('number').value;
             var city = document.getElementById('city').value;
             var add1 = document.getElementById('address').value;
-            var zip = document.getElementById('zip_code').value;
-
+            var zip = document.getElementById('zip_code').value;    
+         let emailregx = new
+      RegExp("^[a-zA-Z0-9\-_]+(\.[a-zA-Z0-9\-_]+)*@[a-z0-9]+(\-[a-z0-9]+)*(\.[a-z0-9]+(\-[a-z0-9]+)*)*\.[a-z]{2,4}$");
```
Comment (engineerscodes) [inline sign-up.html:205]: regex added

File: sign-up.html [hunk 4]
```diff
@@ -212,6 +213,9 @@
             else if(email == ''){
               giveAlert('Please enter your Email address!','info');
             }
+            else if (emailregx.test(email) == false) {
```
Comment (engineerscodes) [inline sign-up.html:216]: Alert message

**Review by nlok5923 — APPROVED** [review]
LGTM!!

**Pull request merged**